ArgusCore/src/main/java/com/salesforce/dva/argus/service/tsdb/DefaultTSDBService.java

```diff
@@ -377,7 +377,17 @@ private ObjectMapper getMapper() {
                 chunkEnd = Math.min(objects.size(), chunkStart + CHUNK_SIZE);
                 try {
                     StringEntity entity = new StringEntity(fromEntity(objects.subList(chunkStart, chunkEnd)));
-                    HttpResponse response = executeHttpRequest(method, endpoint, entity);
+						_logger.debug("{} ({}) -> {}", endpoint, method, new Object() {
```

Comment:
This is really weird looking, and I think your formatting is wrong.  There is a setting in IntelliJ (Editor -> Code Style : Indents Detection.  That will let you use the existing indents without having to change away from the kiwigrid style.